src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1116,12 +1111,6 @@ export default class DataTable extends Vue {
 
     return dataToRender;
   }
-
-  beforeMount() {
-    this._chiMajorVersion = detectMajorChiVersion();
-    this.detectScreenBreakpoint();
```

Comment:
This will break the resize handler functionality and the responsiveness of the data table (hide, show column) will not work.